src/components/AccountCreation/AccountCreation.tsx

```diff
@@ -2,7 +2,18 @@
 // @ts-nocheck
 
 // AccountCreation.tsx
-import React from "react";
+
+import {
+  Box,
+  Button,
+  Container,
+  TextField,
+  ThemeProvider,
+  Typography,
+} from "@mui/material";
+import React, { useState } from "react";
+
+import theme from "./../HomePage/theme";
```

Comment:
We should try to avoid using `..` in paths as much as possible, but we have an alias for the `components` folder that should make this just as easy:

```suggestion
import theme from "@components/HomePage/theme";
```

Reply:
I didn't know we could that! That's good to know.